lib/Util.ts

```diff
@@ -107,17 +107,17 @@ export async function decompressGzippedContent(buffer: Buffer, charset?: string)
  * @return {RegExp}
  */
 export function buildProxyBypassRegexFromEnv(bypass : string) : RegExp {
-    try {
-        // We need to keep this around for back-compat purposes
-        return new RegExp(bypass, 'i')    
-    }
-    catch(err) {
-        if (err instanceof SyntaxError && (bypass || "").startsWith("*")) {            
-            let wildcardEscaped = bypass.replace('*', '(.*)');
-            return new RegExp(wildcardEscaped, 'i');
-        }
-        throw err;
+    const searchRegExpToReplaceSpecialChars: RegExp = new RegExp('(?<!\\\\)([.])(?!\\*)', 'g');
```

Comment:
Nit: could it make sense to move it to constants and not initialize on every call?

Reply:
done